### ISSUE TITLE
Parsing fails with invalid expose protocol

### ISSUE THREAD

**Issue by kaos** [hadolint/hadolint#675]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Lint warning, or at least a syntax error pointing closer to the issue at hand.

### Actual behavior

```
Dockerfile:1:8 unexpected '8' expecting '\', a new line followed by the next instruction, or at least one space 
```

### Steps to reproduce the behavior

Output of `hadolint --version` : `Haskell Dockerfile Linter 2.6.0-no-git`

**Dockerfile**

```Dockerfile
EXPOSE 80/ip
```

**Additional environment details (OS, stack version, etc.)**

MacOS 11.5.1
